trust-store-trust-policy-specification.md

```diff
@@ -181,21 +181,110 @@ The implementation must allow the user to execute custom validations. These cust
+
+##### Revocation Checking with CRL
+
+To check the revocation status of a certificate against CRL, the following steps must be performed:
+
+1. Verify the CRL signature.
+1. Verify that the CRL is valid (not expired). A CRL is considered expired if the current date is after the `NextUpdate` field in the CRL.
+1. Look up the certificate’s serial number in the CRL.
+    1. If the certificate’s serial number is listed in the CRL, look for `InvalidityDate`. If the invalidity date is present and timestamp signature is also present then if the invalidity date is before the timestamping date, the certificate is considered revoked. If the invalidity date is not present in CRL, the certificate is considered revoked.
+    1. If the CRL is expired and the certificate is listed in the CRL for any reason other than `certificate hold`, the certificate is considered revoked.
+    1. If the certificate is not listed in the CRL or the revocation reason is `certificate hold`, a new CRL is retrieved if the current time is past the time in the `NextUpdate` field in the current CRL. The new CRL is then checked to determine if the certificate is revoked. If the original reason was `certificate hold`, the CRL is checked to determine if the certificate is unrevoked by looking for the `RemoveFromCRL` revocation code.
+
+##### Revocation Checking with Delta CRLs
+
+If a delta CRL exists for a base CRL, the `Freshest CRL` extension in the base CRL provides the URL from where the delta CRL can be downloaded. The delta CRLs have their identifying number, plus the "Delta CRL Indicator" extension indicating the version number of the base CRL that must be used with the delta CRL.
+
+When delta CRLs are implemented, the following results can occur during revocation checking.
+
+- If the delta CRL cannot be retrieved for some reason, the revocation result will be "revocation unavailable".
+- If the delta CRL’s indicator is less than the current base CRL, the revocation result returned will be "revocation unavailable"
+- To check revocation of a certificate in Delta CRLs follow steps similar to [Revocation Checking with CRL](#revocation-checking-with-crl).
+
+#### OCSP
+
+##### OCSP Download
+
+OCSP URLs can be obtained from the certificate's authority information access (AIA) extension as defined in [RFC-2560](https://datatracker.ietf.org/doc/html/rfc2560). Notary V2 will wait for a default threshold of 5 seconds to receive an OCSP response. If OCSP response is not available within the timeout threshold the revocation result will be "revocation unavailable". The user may be able to configure this threshold.
```

Comment:
> The user may be able to configure this threshold.

Same comment as above. If we have a way to configure today, let's add a link. If not, we should just remove this sentence and see if/when we need to configure and make that change as a batch pr.

Reply:
So there is no standard here. Basically we are saying that implementation MUST use default threshold of 5 and implementation MAY provide functionality for user to override this values.